index.html

```diff
@@ -8,26 +8,54 @@
   </head>
   <body>
     <script>
-      var user = {
-        "key": "example-user-key",
-        "name": "Sandy"
+    function main()
+    {
+      // Set clientSideID to your LaunchDarkly client-side ID
+      const clientSideID = '';
+
+      // Set flagKey to the feature flag key you want to evaluate
+      const flagKey = 'my-boolean-flag';
+
+      // Set up the user properties. This user should appear on your
+      // LaunchDarkly users dashboard soon after you run the demo.
+      const user = {
+        'key': 'example-user-key',
+        'name': 'Sandy'
       };
 
-      var div = document.createElement("div");
+      var div = document.createElement('div');
       document.body.appendChild(div);
-      
       div.appendChild(document.createTextNode('Initializing...'));
 
-      var ldclient = LDClient.initialize('YOUR_CLIENT_SIDE_ID_HERE', user);
-      
+      if (clientSideID === '') {
+        div.replaceChild(document.createTextNode('Please edit index.html to set clientSideID to your LaunchDarkly SDK key first'), div.firstChild);
+        return;
+      }
+
+      const ldclient = LDClient.initialize(clientSideID, user);
+
+      // TODO: what's the best way to check if this succeeded,
+      // so we can then display "SDK successfully initialized!" or
+      // "SDK failed to initialize" ?
```

Comment:
Is this something that we can do easily, or do we need to leave out? I'm not sure of the best way to check if initialization succeeded.